Automattic/wp-calypso
# Force text color on certain elements rendered in reader on mobile clients

#### Changes proposed in this Pull Request



Currently, reader on mobile clients does not support rendering of colored text. This fix forces text elements to use the platform specific text colors.

Before-After comparisons:

[![Image from Gyazo](https://i.gyazo.com/d1935d01e5b960084c1ee67e675799b9.png)](https://gyazo.com/d1935d01e5b960084c1ee67e675799b9)

[![Image from Gyazo](https://i.gyazo.com/d69148ca76f6b0535d73c22e7614540c.png)](https://gyazo.com/d69148ca76f6b0535d73c22e7614540c)

cc @leandroalonso @gibrown 

#### Testing instructions

It's my first PR towards calypso repo, so I'm not sure how folks test the changes on mobile clients, but:
- Point reader css provider to reader-mobile.css from this PR.
- Android specific: If you use the build-in server in calypso, you can access `reader-mobile.css` from `http://10.0.2.2:3000/calypso/reader-mobile.css`
- Make sure colored text is rendered using default platforms text color in both light and dark modes.






Related to https://github.com/wordpress-mobile/WordPress-Android/pull/13958
